src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs

```diff
@@ -44,13 +44,20 @@ public async Task<ICollection<Dispute>> GetAllDisputesAsync(ExcludeStatus? exclu
 
         foreach(Dispute dispute in disputes)
         {
-            // When reviewing the list of tickets, the JSON is needed to compute SystemDetectedOCRIssues
-            OcrViolationTicket? ocrViolationTicket = null;
-            if (!string.IsNullOrEmpty(dispute.OcrTicketFilename))
+            try
+            {
+                // When reviewing the list of tickets, the JSON is needed to compute SystemDetectedOCRIssues
+                OcrViolationTicket? ocrViolationTicket = null;
+                if (!string.IsNullOrEmpty(dispute.OcrTicketFilename))
+                {
+                    // Retrieve deserialized OCR Violation Ticket JSON Data from object storage for the given filename (NoticeOfDisputeGuid)
+                    ocrViolationTicket = await _filePersistenceService.GetJsonDataAsync<OcrViolationTicket>(dispute.OcrTicketFilename, cancellationToken);
+                    dispute.ViolationTicket.OcrViolationTicket = ocrViolationTicket;
+                }
+            }
+            catch 
             {
-                // Retrieve deserialized OCR Violation Ticket JSON Data from object storage for the given filename (NoticeOfDisputeGuid)
-                ocrViolationTicket = await _filePersistenceService.GetJsonDataAsync<OcrViolationTicket>(dispute.OcrTicketFilename, cancellationToken);
-                dispute.ViolationTicket.OcrViolationTicket = ocrViolationTicket;
+                // dont crash since in dev environment, ocr image could be on individuals localhost

```

Comment:
Just dont swallow the exception. In production, there could be this error and we'll be scratching our heads. Be sure to log out a warning, or use a compiler directive like `#if DEBUG ...`

Reply:
logging exception as is done in getting individual dispute